### PR TITLE
Put a vertical toolbar on the right side

### PR DESCRIPTION
**Pull request by Erlotinib**
This CSS put the personnal Toolbar in a vertical position and place it on the right side of the window.

Use code from Faviconized Bookmarks (Okamoi) to hide favicon name.

### CODE REVIEW

File: toolbars/vertical-toolbar-right-side.css [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  margin-right: 0!important;
+}
+
+#PersonalToolbar {
+  position: fixed!important;
+  /*
+   * We place the toolbar below the url-bar and menu bar
+   * 0 is the top of firefox's interface 
+   */
+  top: 80px!important;
```
[the remark this answers is not in the document]
Comment (Timvde) [reply toolbars/vertical-toolbar-right-side.css]: I also don't know how to do it, sadly. I'm not even a web developer, so my CSS knowledge is quite minimal. In that case, I'd prefer you use a CSS variable that is declared at the top of the file, with a comment. Could you do the same for your other PR?

**Comment by Erlotinib** [comment]
Hello,

I've made the change and added a var at the top of the file with comment.
I hope I didn't mess up history, the interface of gihub is very confusing regarding merge/rebase.

regards

**Comment by Timvde** [comment]
Hmm, something weird went wrong indeed. I'm not sure what you did :P But don't worry, I'll fix it ;) Thanks!

**Comment by Timvde** [comment]
I couldn't find how to push to this branch (maybe I can't, since it is in your fork), so if you could quickly review #106, I'll merge it :) Alternatively, what I did was just a rebase on top of my master. It removed the extra commits. If you do that and push, that's fine for me too.

**Comment by Erlotinib** [comment]
merged with #106

**Pull request closed without merging**